src/sider/manglendemedvirkning/stans/StansSkjema.tsx

```diff
@@ -59,27 +66,42 @@ interface Props {
 
 export default function StansSkjema({ varselSvarfrist }: Props) {
   const personident = useValgtPersonident();
-  const sendVurdering = useSendVurderingManglendeMedvirkning();
-  const formMethods = useForm<StansSkjemaValues>();
+  const sendVurdering = useSendVurdering<StansVurdering>();
   const {
     register,
     watch,
+    setValue,
+    getValues,
     formState: { errors },
+    setError,
+    clearErrors,
     handleSubmit,
-  } = formMethods;
+  } = useForm<FormValues>();
 
-  const { getStansDocument } = useManglendeMedvirkningVurderingDocument();
+  const { datepickerProps, inputProps } = useDatepicker({
+    fromDate: varselSvarfrist,
+    onDateChange: (date: Date | undefined) => {
+      if (!!date) {
```

Comment:
Trenger man håndtere setting og clearing av errors her? 
Ser ikke ut som vi gjør det i `AvslagDatePicker` og `VedtakFraDato` hvor vi også har en påkrevd dato. Hadde vært nice å slippe kode for det her også, kanskje det er lettere om man skiller det ut som egen komponent? :)

Reply:
Er litt redd for controlled components ettersom jeg ikke helt forstår hvordan det fungerer, men var ikke så mye greier så måtte til ser jeg. Endrer 🚀

Reply:
Synes det blir litt vel magisk med `FormProvider`, men fant ikke en alternativ måte, så da blir det det 🥲

Reply:
Blir bra det :+1: